@xen-orchestra/web/src/components/host/HostPifsTable.vue

```diff
@@ -0,0 +1,257 @@
+            size="medium"
+          >
+            {{ $t('edit') }}
+          </UiButton>
+          <UiButton
+            v-tooltip="$t('coming-soon')"
+            disabled
+            :left-icon="faTrash"
+            variant="tertiary"
+            accent="danger"
+            size="medium"
+          >
+            {{ $t('delete') }}
+          </UiButton>
+          <template #title>
+            <UiActionsTitle> {{ $t('table-actions') }}</UiActionsTitle>
+          </template>
+        </UiTableActions>
+
+        <UiTopBottomTable :selected-items="0" :total-items="0" />
+      </div>
+      <VtsDataTable :is-ready :has-error :no-data-message="pifs.length === 0 ? $t('no-network-detected') : undefined">
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th v-if="column.id === 'checkbox'" v-tooltip="$t('coming-soon')" class="checkbox">
+                <UiCheckbox :v-model="areAllSelected" accent="info" disabled />
+              </th>
+              <th v-else-if="column.id === 'more'" class="more">
+                <UiButtonIcon v-tooltip="$t('coming-soon')" :icon="faEllipsis" accent="info" disabled size="small" />
+              </th>
+              <th v-else>
+                <div v-tooltip class="text-ellipsis">
+                  <VtsIcon accent="brand" :icon="headerIcon[column.id]" />
+                  {{ column.label }}
+                </div>
+              </th>
+            </template>
+          </tr>
+        </template>
+        <template #tbody>
+          <tr
+            v-for="row of rows"
+            :key="row.id"
+            :class="{ selected: selectedPifId === row.id }"
+            @click="selectedPifId = row.id"
+          >
+            <td
+              v-for="column of row.visibleColumns"
+              :key="column.id"
+              class="typo p2-regular"
+              :class="{ checkbox: column.id === 'checkbox' }"
+            >
+              <div v-if="column.id === 'checkbox'" v-tooltip="$t('coming-soon')">
+                <UiCheckbox v-model="selected" accent="info" :value="row.id" disabled />
+              </div>
+              <div v-else-if="column.id === 'status'" class="status">
+                <PifStatus :network="getNetwork(row.value)" />
+              </div>
+              <UiButtonIcon
+                v-else-if="column.id === 'more'"
+                v-tooltip="$t('coming-soon')"
+                :icon="faEllipsis"
+                accent="info"
+                disabled
+                size="small"
+              />
+              <div v-else v-tooltip="{ placement: 'bottom-end' }" class="text-ellipsis">
+                <UiComplexIcon v-if="column.id === 'network'" size="medium" class="icon">
+                  <VtsIcon :icon="faNetworkWired" accent="current" />
+                  <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+                </UiComplexIcon>
```

Comment:
Since the `UiComplexIcon` needs to be reworked, and there is no page to link to the network detail, you can only display the name of the network.